recipes/libunistring/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibUnistringConan(ConanFile):
+    name = "libunistring"
+    description = "This library provides functions for manipulating Unicode strings and for manipulating C strings according to the Unicode standard."
+    homepage = "https://www.gnu.org/software/libunistring/"
+    topics = ("conan", "libunistring", "unicode", "string")
+    license = "LGPL-2.1"
```

Comment:
```suggestion
    license = "LGPL-3.0-or-later"
```

From README file:

> The libunistring library and its header files are dual-licensed under
"the GNU LGPLv3+ or the GNU GPLv2".